src/components/room/Room.tsx

```diff
@@ -49,11 +54,36 @@ const Room = () => {
       },
       [MessageTypes.ROOM_DELETED]: (props: any) => {
         removeCookie('userId');
+        removeCookieIsHost('isHost');
         navigate(`/`);
       }
     };
   }, []);
 
+  // Used when board loads until player square contains player with an id.
+  // Then changes isPlayerHost value to true when:
+  // 1) Targeted square id matches current player.
+  // OR
+  // 2) Cookie contains information that player is host.
+  useEffect(() => {
```

Comment:
how about reconnect doe

Reply:
> how about reconnect doe

during reconnect check if cookie exists

Reply:
> and isBoardLoaded is used bcs at first the square is empty so it would always be false

why would the square be empty at first? When player gets `JOIN_ROOM` the board is already updated